build.js

```diff
@@ -618,16 +619,91 @@ module.exports = function (gulpWrapper, ctx) {
         gulpWrapper.seq(["__internal-build"], callback);
     });
 
+    /**
+     * Build Documentation Database(s) for Search task
+     */
+    gulp.task('__build-db', function (callback) {             
+        if (ctx.type === "documentation") {
+            var elasticlunr = require("elasticlunr");
+    
+            var documentsIndex = elasticlunr(function () {
+                this.addField("id");
+                this.addField("title")
+                this.addField("text");                     
+                this.saveDocument(false);
+            });
+        
+            var documentsList = [];
+        
+            glob(path.join(ctx.baseDir, "./assets/**/*.md"), function (err, matches) {
```

Comment:
I fear that if assets folder doesn't exist, this will throw an error. The code should create one if it doesn't exist to avoid unnecessary errors.

Reply:
Done